tests/fixtures/anim/targets/delta/MGV-GENOME-0264574_vs_OP073605.delta

```diff
@@ -1,2 +1,8 @@
-/Users/pc40583/repositories/pyani-plus/tests/generate_fixtures/../fixtures/viral_example/MGV-GENOME-0264574.fna /Users/pc40583/repositories/pyani-plus/tests/generate_fixtures/../fixtures/viral_example/MGV-GENOME-0357962.fna
+/Users/pc40583/repositories/pyani-plus/tests/generate_fixtures/../fixtures/viral_example/MGV-GENOME-0264574.fna /Users/pc40583/repositories/pyani-plus/tests/generate_fixtures/../fixtures/viral_example/OP073605.fasta
 NUCMER
+>MGV_MGV-GENOME-0264574 OP073605.1 39253 57793
+107 39253 57793 18644 27 27 0
+-2991
+-24624
+-1
+0
```

Comment:
GitHub is showing the overall diff for the pull request as renaming and editing this file.

As per the history, we delete the old file (which had no alignment), and add a new files (which does have an alignment).